EIPS/eip-858/calculations.md

```diff
@@ -0,0 +1,24 @@
+| Variable           | Symbol       | Value         | Unit          | Source |
+| -------------------|--------------|---------------|---------------|--------|
+| Network Hashrate   |H<sub>N</sub> | 232001        | GH/s          | https://etherscan.io/chart/hashrate |
+| Power/Hashrate     |P<sub>H</sub> | 3.54          | W*s/MH        | http://www.legitreviews.com/geforce-gtx-1070-ethereum-mining-small-tweaks-great-hashrate-low-power_195451, https://www.reddit.com/r/ethereum/comments/7vewys/10000_tons_co2_per_day_and_climbing_eip_858/dtrswyz/ |
```

Comment:
What does `Ws/MH` mean? `Ws` is meaningful as the power usage per second at full load. `W/MH` is meaningful as the power required for a certain hash rate. How do you define `Ws/MH` should

Reply:
Maybe I'm mixing up some units though, feel free to correct me.

Reply:
It's a ratio of the hashrate (MH/s) to the power achieving that hashrate consumes, but inversed. You often see GPUs rated as the hashrate achieved at a certain power draw (ie. 31.2 MH/s @ 110.6 W) so this combines that into one metric for simplicity. 

I think I will not invert it so that that's a little clearer.

Reply:
Or better yet, just keep them separate.